code/modules/industrial_lift/elevator/elevator_music_zone.dm

```diff
@@ -0,0 +1,108 @@
+GLOBAL_LIST_EMPTY(elevator_music)
+
+/obj/effect/abstract/elevator_music_zone
```

Comment:
this should probably be a component not an abstract effect

Reply:
or was this done for mapper control?

Reply:
This was done so that mappers can more easily control the area the music plays in yes, previously it was assigned to the elevator panel object which tends to be at the edge of the required area rather than central.